content/news/2023-10-21-bevy-0.12/index.md

```diff
@@ -805,13 +805,37 @@ Bevy uses the [tracing](https://crates.io/crates/tracing) library to measure sys
 
 ![tracing overhead](tracing-overhead-reduction.png)
 
+## Pitch Audio Source
+
+<div class="release-feature-authors">authors: @basilefff</div>
+
+Audio can now be played by pitch, which is useful to debug audio issues, use as a placeholder or for programmatic audio.
```

Comment:
```suggestion
Audio can now be played by pitch, which is useful to debug audio issues, to use as a placeholder, or for programmatic audio.
```

Reply:
I would prefer to not use "simple" as that's a value call and people always do surprising things